### PR TITLE
[DOCS] Adds docstring for expect_column_distinct_values_to_contain_set

### PR DESCRIPTION
**Pull request by rdodev**
Changes proposed in this pull request:
- Adds doctring for config check for `expect_column_distinct_values_to_contain_set`

### CODE REVIEW

File: great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py [hunk 1]
```diff
@@ -101,7 +101,21 @@ class ExpectColumnDistinctValuesToContainSet(ColumnExpectation):
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
-        """Validating that user has inputted a value set and that configuration has been initialized"""
+        """Validates the configuration for the expectation.
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py]: Capitalize Expectation

File: great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py [hunk 2]
```diff
@@ -101,7 +101,21 @@ class ExpectColumnDistinctValuesToContainSet(ColumnExpectation):
...
     ) -> None:
-        """Validating that user has inputted a value set and that configuration has been initialized"""
+        """Validates the configuration for the expectation. 
+        
+        For `expect_column_distinct_values_to_contain_set`
+        we require that the `configuraton.kwargs` contain a `value_set` key that is either a `list`, `set`,
+        or `dict`.
+
+        Args:
+            configuration: The configuration to be validated.
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py]: Propose: "The `ExpectationConfiguration` to be validated."

File: great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py [hunk 3]
```diff
@@ -101,7 +101,21 @@ class ExpectColumnDistinctValuesToContainSet(ColumnExpectation):
...
+
+        Returns:
+            `None` if the configuration is valid
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py]: This always returns `None`. I think we can remove this block.

File: great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py [hunk 4]
```diff
@@ -101,7 +101,21 @@ class ExpectColumnDistinctValuesToContainSet(ColumnExpectation):
...
+            
+        Raises:
+            InvalidExpectationConfigurationError: The configuraton does not contain the values required by the expectation
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py]: Is this > 120 chars?

Comment (NathanFarmer) [reply great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py]: Capitalize Expectation

**Review by NathanFarmer — DISMISSED** [review]

**Review by Shinnnyshinshin — APPROVED** [review]
LGTM. Thank you @rdodev !

**Pull request merged**